DevelopersGuide.adoc

```diff
@@ -616,7 +616,7 @@ Create a `deps.edn` file with this content:
            com.fulcrologic/fulcro {:mvn/version "3.0.10"}}
 
  :aliases {:dev {:extra-paths ["src/dev"]
-                 :extra-deps  {org.clojure/clojurescript   {:mvn/version "1.10.520"}
+                 :extra-deps  {org.clojure/clojurescript   {:mvn/version "1.10.1.10.742"}
                                thheller/shadow-cljs        {:mvn/version "2.8.40"}
```

Comment:
Wouldn't you also need to bump shadow???

Reply:
```{:paths ["src/main" "resources"]
 :deps {org.clojure/clojure    {:mvn/version "1.10.1"}
        com.fulcrologic/fulcro {:mvn/version "3.0.10"}}
 :aliases {:dev {:extra-paths ["src/dev"]
                 :extra-deps {org.clojure/clojurescript {:mvn/version "1.10.742"}
                              thheller/shadow-cljs      {:mvn/version "2.8.107"}
                              binaryage/devtools        {:mvn/version "0.9.10"}}}}}```

That's my working deps.edn. I can try with latest shadow too.